src/samples/__tests__/e2e.test.ts

```diff
@@ -139,19 +136,21 @@ class PollingInboundTransporter implements InboundTransporter {
   }
 
   private pollDownloadMessages(agent: Agent) {
-    poll(
-      async () => {
+    const sleep = (ms: number) => new Promise(res => setTimeout(res, ms));
```

Comment:
I see the same function in `src/lib/__tests__/helpers.ts` maybe we could re-use it

Reply:
I don't think it's a problem that samples depends on lib tests. Updated